recipe/build.sh

```diff
@@ -324,12 +324,14 @@ Mingw_w64_makefiles() {
     else
       mkdir miktex || true
       pushd miktex
-      MIKTEX_VER=2.9.6942
+      MIKTEX_VER=2.9.7100
       # Fetch e.g.:
       # http://ctan.mines-albi.fr/systems/win32/miktex/tm/packages/url.tar.lzma
       # http://ctan.mines-albi.fr/systems/win32/miktex/tm/packages/mptopdf.tar.lzma
       # http://ctan.mines-albi.fr/systems/win32/miktex/tm/packages/inconsolata.tar.lzma
-        curl --insecure -C - -o ${DLCACHE}/miktex-portable-${MIKTEX_VER}.exe -SLO https://miktex.org/download/ctan/systems/win32/miktex/setup/windows-x86/miktex-portable-${MIKTEX_VER}.exe || true
+        # From https://miktex.org/download#portable :
+        #   Please note that there is no seperate installer. Just download the standard installer and rename it to miktex-portable.exe.
+        curl --insecure -C - -o ${DLCACHE}/miktex-portable-${MIKTEX_VER}.exe -SL https://miktex.org/download/ctan/systems/win32/miktex/setup/windows-x86/basic-miktex-${MIKTEX_VER}.exe || true
```

Comment:
@conda-forge/r-base
- I'm a monkey and don't know what I'm doing here, so PTAL ;)
- The old URL didn't work, so I updated it.
- Removed the `-O` flag since it's redundant with `-o-`.
- The old `miktex-portable-${MIKTEX_VER}.exe` URL scheme doesn't work anymore; now it's just `basic-miktex-${MIKTEX_VER}.exe` or the unversioned `miktex-portable.exe`.
- (Temporarily?) Copied the the note from their website to avoid future head scratches about that.

Reply:
Well, turns out a simple `7za x` doesn't cut it anymore. Now the installer contains an archive with still packed components. Running the installer with `--portable` should do it. See https://docs.miktex.org/manual/setupwiz.html.